### PR TITLE
Add endpoint for retrieving Canvas submissions

### PR DESCRIPTION
**Pull request by westonkd**
_Fixes PLAT-2302_
Test Plan:
Wait for a submission created webhook to be received and then navigate to
```
http://originality.docker/tool_proxy/:tool_proxy_guid/submissions/:tc_id/retrieve
```
Verify the submission in the tool is updated with the attachment data from canvas.

### CODE REVIEW

File: app/models/originality_report.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class OriginalityReport < ApplicationRecord
-  validates :originality_score, :file_id, :tc_id, :submission, presence: true
+  validates :originality_score, :file_id, :file_id, :tc_id, :submission, presence: true
```
Comment (rivernate) [inline app/models/originality_report.rb]: it looks like there is a duplicate `:file_id`

File: app/controllers/submissions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# SubmissionsController
```
Comment (rivernate) [inline app/controllers/submissions_controller.rb]: I think we have a SubmissionController already, we should probably rename it to plural, and add this in.

That's my bad I named it wrong

File: app/models/originality_report.rb (hunk 1 above)
Comment (westonkd) [inline app/models/originality_report.rb]: done

File: app/controllers/submissions_controller.rb (hunk 2 above)
Comment (westonkd) [inline app/controllers/submissions_controller.rb]: No worries, I just finished pluralizing

**Pull request merged**